lib/net.js

```diff
@@ -1326,84 +1326,71 @@ function listen(self, address, port, addressType, backlog, fd, exclusive) {
 
 
 Server.prototype.listen = function() {
-  var self = this;
+  const argsLen = arguments.length;
```

Comment:
I just copied this snippet; it was introduced in https://github.com/nodejs/node/commit/d0582ef9e19e8ed941b0a585c935ad11919080ee. If you want to change this, I'd say it should be changed everywhere, and I think that doesn't fit into this PR.